sycl/test/basic_tests/host_platform_avail.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clangxx -fsycl %s -o %t1.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUN: %GPU_RUN_PLACEHOLDER %t1.out
+// RUN: %ACC_RUN_PLACEHOLDER %t1.out
+
+//==------ host_platform_avail.cpp - Host Platform Availability test -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+class foo;
+int main() {
+  queue q;
+  auto dev = q.get_device();
+  auto dev_platform = dev.get_platform();
+
```

Comment:
Why do you need that? I think the body of the if on line 25 should be enough for the test.

Reply:
Good point.